packages/search/README.md

```diff
@@ -129,29 +129,6 @@ Map model
   });
 ```
 
-- Connect to Crafter Search to query for content with SOLR (crafter version: 3.0.x):
-
-```typescript
-  import { crafterConf } from '@craftercms/classes';
-  import { search, createQuery } from '@craftercms/search';
-
-  //First, set the Crafter configuration to _cache_ your config. 
-  //All subsequent calls to `getConfig` will use that configuration.
-  crafterConf.configure({
-    baseUrl: 'http://localhost:8080',
-    site: 'editorial',
-    searchId: 'editorial'   // if searchId is the same as site, this parameters is not needed
-  })
-
-  const query = createQuery('solr');
-  query.query = "*:*";
-  query.filterQueries = ['content-type:"/component/video"'];
-
-  search(query).subscribe((results) => {
-    // ...
-  });
-```
-
 You may alternatively use a different config by supplying the config object at the service call invoking time
```

Comment:
This paragraph all alone doesn't make sense.

Reply:
That refers to the ability to set the configuration separately (first example) or sending it as the second parameter in `search`. It feels that it's still valid.